applications/crossbar/priv/couchdb/schemas/system_config.tasks.json

```diff
@@ -21,6 +21,16 @@
             "minimum": 1,
             "type": "integer"
         },
+        "compaction_list_dbs_chunk_size": {
+            "default": 20,
+            "description": "tasks compaction_list_dbs_chunk_size",
```

Comment:
Can we please start adding real descriptions, not use the placeholder auto-generated ones?